service-users-specs/src/test/resources/scenarios/userRetrival.feature

```diff
@@ -0,0 +1,35 @@
+Feature: User Retrieval
+
+  Background:
+    Given there is a User and Team Server
```

Comment:
hum... does that mean that you generate test data? If that is the case, you should state it in the Background, writing something like "And test data has been generated" (ideally with a table).

Reply:
Using this test , I verify if the GET list is a list of 4 users. 
I test the first one on the background, so if there is an error, the other tests will not be tested. I did that because Cucumber will sort files according to their file name before running them.